locale/pl/LC_MESSAGES/client.po

```diff
@@ -95,7 +95,7 @@ msgstr "Przykro nam o tym słyszeć. Jeśli nie ufasz już temu urządzeniu, to
 
 #: .es5/scripts/views/settings/clients.js:52
 msgid "Last active %(translatedTimeAgo)s"
-msgstr ""
+msgstr "Ostatnia aktywność: %(translatedTimeAgo)s"
```

Comment:
Yea this is a stop gap solution, we have several problems with current string extraction, such as plural forms, extra info about the string, etc. Hopefully we can gather those issues together and develop a better way to manage all these strings in the next couple of months.